src/lib/components/layout/index.scss

```diff
@@ -9,6 +9,8 @@
   --right-drawer-width: 300px;
 }
 
+$contentSlot: ":not(.q-header, .q-footer, .q-railbar, .q-drawer)";
```

Comment:
Do we want to use `camelCase` or `kebab-case` for SCSS vars? So far I've used the latter, but I'm open to discuss it.